packages/customize-widgets/src/components/test/error-boundary.js

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
The unit-tests are identical but simple enough. We can turn them into a single test once we work on https://github.com/WordPress/gutenberg/pull/42024#discussion_r909341655.

Reply:
Since we plan on working on https://github.com/WordPress/gutenberg/pull/42024#discussion_r909341655, I'll abstain from trying to DRY the unit-tests (by extracting the common logic into a shared function or something along those lines, for example).